apps/fabric-website-resources/test/FocusZone.test.dom.ts

```diff
@@ -0,0 +1,42 @@
+import * as puppeteer from 'puppeteer';
+
+describe('FocusZone', () => {
+  let browser: any;
+  let page: any;
+
+  beforeAll(async () => {
+    browser = await puppeteer.launch({ headless: true });
+    page = await browser.newPage();
+    await page.goto('http://localhost:4322/#examples/focuszone');
```

Comment:
Could this maybe instead just be a helper like `renderPage(content: JSX): Promise<{ browser, page }>` ?

There is an implicit dependency on this page now, and it's going to be really fragile. I'd love instead to have something self contained. Sent mail, that might already be the plan but wanted to get clarification.